plugins/com.google.cloud.tools.eclipse.appengine.newproject.maven/src/com/google/cloud/tools/eclipse/appengine/newproject/maven/MavenArchetypeProjectWizard.java

```diff
@@ -0,0 +1,65 @@
+
+package com.google.cloud.tools.eclipse.appengine.newproject.maven;
+
+import com.google.cloud.tools.eclipse.appengine.newproject.AppEngineStandardWizardPage;
+
+import org.eclipse.core.runtime.IProgressMonitor;
+import org.eclipse.core.runtime.IStatus;
+import org.eclipse.core.runtime.Status;
+import org.eclipse.jface.operation.IRunnableWithProgress;
+import org.eclipse.jface.viewers.IStructuredSelection;
+import org.eclipse.jface.wizard.Wizard;
+import org.eclipse.ui.INewWizard;
+import org.eclipse.ui.IWorkbench;
+import org.eclipse.ui.statushandlers.StatusManager;
+import org.slf4j.Logger;
```

Comment:
Any string reason to add this dependency? Google-standard (which doesn't necessarily apply here on Github) is to use java.util.logging